docs/4.3/installation.md

```diff
@@ -53,8 +53,8 @@ up-to-date information.
 === "Amazon Linux 2"
 
     ```bash
-    $ wget https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-    $ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    $ wget https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm
+    $ tar -xzf teleport-v{{ teleport.version }}-1.x86_64.rpm
```

Comment:
```suggestion
    $ yum install https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm
```

Reply:
`curl` and `sha256sum` aren't available on AWS boxes by default.

Reply:
OK, as there's no default way to check the checksum anyway then how about we just install direct with `yum`? I'll change the suggestion appropriately.

Reply:
We could also `wget` and then `rpm -i` if that's more consistent with the other RPM method we list.

Reply:
Yes, please change the suggestion to proposed `yum` based flow.